### DISCUSSION TITLE
NVME Speed drops to 85% on Dell T5810

### DISCUSSION THREAD

**Discussion by dslrnation** [Chia-Network/chia-blockchain#5483] [Q&A]
Hi guys, I have been testing many nvme Drives, and they start very fast when I create 3 plots at the same time, on task manager the drive speed reaches up to 1.2GB while plotting at 1% and then the speed drops to 170MB. The first Phase of plotting is taking about 18500 Seconds

**I'm Using :

Dell Precision T5810 Tower
Xeon E5-2670 v3 2.3ghz 12 Cores
24 Threads
8x 4GB 2133 DDR4 = 32GB
160GB Solid State for OS
Seagate 8TB HDD for farming

**For plotting I tested**
Firecuda 520 2TB nvme
Inland platinum 1TB nvme
SK hynix Gold P31 1TB nvme

All nvme drives > tested with a pci-e adapter

I know nvme drives drops after a while but 185000 seconds for the first phase is too much, I will test the Samsung 980 pro 1tb nvme when I receive it

What am I doing wrong ?

Thanks in advance Farmers

**Comment by T4C-FanBoy** [comment]
Same for me with the wd black SN850 in gen4. I guess its normal.

**Comment by SackofFluor** [comment]
SSD Buffer Full?
PCie Slot too slow?

SSD Overheating?
